3p/ampanalytics-lib.js

```diff
@@ -0,0 +1,264 @@
+/**
```

Comment:
Can do, if you think this will be more meaningful to customers. (Note that I am taking a vacation day Tuesday 7/25.)